tests/api/test_dash_acl_group.py

```diff
@@ -0,0 +1,26 @@
+from pprint import pprint
+
```

Comment:
This file duplicates similar in https://github.com/opencomputeproject/SAI-Challenger/pull/195/files

Reply:
removed dash case